web/src/reducers/appReducer.js

```diff
@@ -375,12 +375,18 @@ const reducer = (state = INITIAL_STATE, { type, payload = {} }) => {
 				if (pairs.includes(key)) {
 					let temp = state.tickers[key] || {};
 					let pairTrade = payload[key][0];
-					let close =
-						pairTrade && pairTrade.price
-							? pairTrade.price
-							: temp.close
-							? temp.close
-							: 0;
+					let close = (
```

Comment:
Seems that we are mutating the previous state here. I dont think its going to affect the app functionality since we are finally creating a copy of the state, but I think it may cause issues in case we are going to debug or use redux time travel.